src/renderer/qssWidget/js/qssWidget.js

```diff
@@ -71,6 +71,7 @@
         events: {
             onWidgetClosed: null,
             onQssWidgetHideQssRequested: null,
+            onQssOpenUsbRequested: null,
```

Comment:
My suspicion is that these updates to qssWidget.js are unnecessary - could you experiment with removing them and report what happens?

Reply:
Yup, you are totally right about this, it works without the definition here.